exercises/concept/log-levels/src/test/java/LogLevelsTest.java

```diff
@@ -1,62 +1,86 @@
-import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.*;
 
 public class LogLevelsTest {
     @Test
+    @Tag("task:1")
+    @DisplayName("The message method method returns the log line's message of an error log")
```

Comment:
Typo: “method method” should probably be “method” 😁 goes for all task 1&2 tests fyi